ui/src/lib/gql/document/useQueries.ts

```diff
@@ -68,6 +68,25 @@ export const useQueries = () => {
       refetchOnWindowFocus: false,
     });
 
+  const usePreviewSnapshot = useMutation({
```

Comment:
Yet again I'm getting mixed up. How is this different from useProjectSnapshotQuery? It being a mutation is confusing.
Whichever it is, the name should meet the appropriate naming convention